code/modules/mob/living/carbon/xenomorph/castes/wraith/abilities_wraith.dm

```diff
@@ -476,7 +476,9 @@ GLOBAL_LIST_INIT(wraith_banish_very_short_duration_list, typecacheof(list(
 			if(ishuman(pulled_target))
 				var/mob/living/carbon/human/H = pulled_target
 				if(H.stat == UNCONSCIOUS) //Apply critdrag damage as if they were quickly pulled the same distance
-					H.adjustOxyLoss(HUMAN_CRITDRAG_OXYLOSS * get_dist(H.loc, T))
+					var/critdamage = HUMAN_CRITDRAG_OXYLOSS * get_dist(H.loc, T)
+					if(!H.adjustOxyLoss(critdamage))
```

Comment:
why are you not using a species flag here?

Reply:
General solution if say we add a nooxy implant/chem or such it stops bodydrag from being an issue while still punishing